src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -44,6 +44,10 @@ $description--margin-top: 2px;
     }
   }
 
+  &--wide {
+    width: var(--event--widen) !important;
```

Comment:
Please get rid of the `!important` it's a bad practice.

Reply:
done fd981d9